### DISCUSSION TITLE
v3.0-(1402-5-26)

### DISCUSSION THREAD

**Discussion by GFW-knocker** [GFW-knocker/MahsaNG#16] [Announcements]
ابتدا نسخه قبلی را حذف و سپس اقدام به نصب نسخه جدید کنید 

برای گوشی های جدید از نسخه
v8a
و برای گوشی های قدیمی تر از 
v7a
استفاده کنید 
مدل پردازنده گوشی را در پایین منوی تنظیمات تلگرام میتوانید ببینید
همچنین میتوانید از نسخه
generic
برای هر مدل گوشی استفاده کنید

[کانال تلگرام برنامه](https://t.me/mahsa_net)

اگر به برنامه اعتماد ندارید آن را در سایت
[virustotal.com](https://www.virustotal.com/gui/home/url)
با تمامی آنتی ویروس های روز دنیا چک کنید
همین نسخه بزودی در گوگل پلی نیز منتشر میشود

add lots of powerful features including intelligent config distribution
add Config Donation menu and website
add strong security features
fix core dns issue
fix ui bugs

<hr /><em>This discussion was created from the release <a href='https://github.com/GFW-knocker/MahsaNG/releases/tag/v3.0-(1402-5-26)'>v3.0-(1402-5-26)</a>.</em>

**Comment by mimpzk** [comment]
نسخه ios اپ رو نمیسازید؟🥲
